### DISCUSSION TITLE
How do I remove a user

### DISCUSSION THREAD

**Discussion by ArcticWarmth** [NamelessMC/Nameless#2019] [Q&A] [answered]
I was wondering how to remove a user.

**Comment by samerton** [comment, marked as answer]
There's a bug in pre-release 8 where the option is missing.

Pre-release 9 should be out shortly which will include a fix - once that's done, when you edit a user in the StaffCP -> User Management -> Users tab there will be an option to delete the user in the Actions dropdown.

**Reply by ArcticWarmth** [reply]
That's what I thought I had to do. Thanks for letting me know about this bug. :)
